docs/dapps/nfts/preparing-nft-files.md

```diff
@@ -0,0 +1,88 @@
+# Preparing NFT Files for an ERC721 Smart Contract
+
+The first step of setting up an NFT smart contract is having your NFT files ready to use. In this example, the files will get uploaded to [Pinata](https://www.pinata.cloud/), a pinning service that prevents files from being garbage collected on IPFS. If you do not already have an account, please create one.
```

Comment:
Consider using contractions to make the text read more casually.

Ex: "If you don't already..." instead of "If you do not already..."